Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/LightweightStylingSamplePage.xaml.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.Theming, "Lightweight Styling", Description = "Lightweight styling enables the customization of a control's appearance by adjusting individual keys rather than completely redefining its style.", DocumentationLink = "https://material.io/components/cards")]
```

Comment:
DocumentationLink is incorrect here @silviuo

Reply:
hi @agneszitte , is this the correct link?

https://learn.microsoft.com/en-us/windows/apps/design/style/xaml-styles#lightweight-styling

Thanks!